config/locales/en.yml

```diff
@@ -383,7 +383,7 @@ en:
         email: Business e-mail
         username: Contact username
       max_toot_chars:
-        desc_html: Set maximum toot length in characters (default: 500)
+        desc_html: "Set maximum toot length in characters (default: 500)"
```

Comment:
It seems that the linter is complaining about the use of double quotes here instead of single quotes-- if you change to single quotes the it should pass the tests.

Reply:
done